cfct/manifest.yaml

```diff
@@ -0,0 +1,121 @@
+# This is a sample, non-production-ready file
+# © 2021 Amazon Web Services, Inc. or its affiliates. All Rights Reserved.  
+# This AWS Content is provided subject to the terms of the AWS Customer Agreement available at  
+# http://aws.amazon.com/agreement or other written agreement between Customer and either
+# Amazon Web Services, Inc. or Amazon Web Services EMEA SARL or both.
+
+---
+#Default region for deploying Custom Control Tower: Code Pipeline, Step functions, Lambda, SSM parameters, and StackSets
+region: ap-southeast-2
+version: 2021-03-15
+
+# Control Tower Custom CloudFormation Resources
+resources: 
+
+  - name: VPCFlowLogsBucket
+    resource_file: templates/vpc_flowlog_logging_bucket.template
+    parameters:
+      - parameter_key: "OrganizationId"
+        parameter_value: "$[alfred_ssm_/org/core/OrganizationID]"
+
+      - parameter_key: "SSEAlgorithm"
+        parameter_value: "AES256"
+
+      - parameter_key: "KMSMasterKeyID"
+        parameter_value: ""
+    deploy_method: stack_set
+    deployment_targets:
+      accounts:
+        - Audit
+    export_outputs:
+      - name: /org/sharedservice/networking/vpcflowlogsbucket
+        value: $[output_oBucketName]
+    regions:
+      - ap-southeast-2
+
+  - name: VPCFlowLogsMgmtRole
+    resource_file: templates/vpc_flowlog_mgmt.template
+    parameters:
+      - parameter_key: "OrganizationId"
+        parameter_value: "$[alfred_ssm_/org/core/OrganizationID]"
```

Comment:
this SSM param does not exist by default, I suggest to make call out in the README

Reply:
I added the management_prep.template which adds this param in, along with the AWSControlTowerExecution role